plugins/woocommerce/client/legacy/css/_mixins.scss

```diff
@@ -383,3 +383,22 @@
 		width: 1px;
 	}
 }
+
+@mixin woocommerce-float-label {
```

Comment:
Create a mixin with the float label style to be used by the Twenty* themes stylesheets.